mwui-stencil/src/components/mw-button/mw-button.scss

```diff
@@ -1,3 +1,7 @@
+a {
+  text-decoration: none;
+}
+
 @mixin button-sizes {
```

Comment:
It's not your changes causing it, but probably we need to treat a button without label but with one icon differently at the paddings. I think it should be a circle but it's a pill. Totally as expected but maybe we could create an explicit IconButton for that.